examples/filters/sensorfilter/sensor_filter.go

```diff
@@ -90,10 +90,11 @@ func relativeChange(curr, prev map[string]interface{}, logger golog.Logger) floa
 		logger.Errorw("sensor's previous distance reading is not of type float", "prevReading", prev)
 		return 0
 	}
-	diff := currDist - prevDist
+	logger.Warnln("calculating rel change - curr: ", currDist, "; prev: ", prevDist)
 	if prevDist == 0 {
-		return math.Abs(diff)
+		return math.Abs(currDist)
```

Comment:
I think this is a little weird and mathematically incorrect. As an exercise, consider if you have (9 - 1e-34)/ (1e-34), where the denominator is very close to 0. That actually equals a very large number, where we would be returning 9. I think we should use the IEEE divide by zero rules [here](https://en.wikipedia.org/wiki/Division_by_zero#:~:text=In%20IEEE%20754%20arithmetic%2C%20a,when%20dividing%20by%20%E2%88%920%20instead.&text=The%20justification%20for%20this%20definition,in%20case%20of%20arithmetic%20underflow.) to decide what to do when the denominator is a 0. Open to other suggestions!

Reply:
Hmm gotcha - since this code will be used as an example for users to build filter components, I want to keep the filter logic as simple and digestible as possible. Switching to delta instead of relative change!